internal/encoding/ssh/filexfer/openssh/posix-rename.go

```diff
@@ -4,54 +4,54 @@ import (
 	sshfx "github.com/pkg/sftp/internal/encoding/ssh/filexfer"
 )
 
-const extensionPosixRename = "posix-rename@openssh.com"
+const extensionPOSIXRename = "posix-rename@openssh.com"
```

Comment:
`POSIX` is an all-caps abbreviation that should be all-caps in variable names. (Are you glad we had this all in `internal` so we could change this bad early draft choice? :trollface: )